### ISSUE TITLE
Feature Request: FTP option for library

### ISSUE THREAD

**Issue by Celoain** [cadon/ARKStatsExtractor#444]
I wonder if it'd be possible to have the library on a FTP server and being able to use it with multiple clients?

**Comment by cadon** [comment]
Hi, you can do it with a cloud-service, see https://github.com/cadon/ARKStatsExtractor/wiki/FAQ#how-can-i-share-the-list-with-my-tribe-members-and-edit-simultaneosly

An FTP-server is not yet supported.

**Comment by defyreality** [comment]
i do this with onedrive. would also work with pretty much any other cloud drive service. dropbox, mega, etc...

**Comment by Celoain** [comment]
Yeah, i didn't see that feature. Though it's not really as easy as a simple FTP server for the end-user. Everyone is on different Cloudsystems, sharing them with everyone is going to get pretty hard.

I might end up having some spare time, would you incorporate a FTP-System if it was provided in a PR?

**Comment by cadon** [comment]
I've no experience with FTP yet, if you have and make a pull-request, I can include it in the releases. You can also contact me on discord (cadaei#4308) if you have further questions.
